src/RuleSet/RuleSet.php

```diff
@@ -222,34 +223,34 @@ public function getRulesAssoc($mRule = null)
      * Note: this is different from pre-v.2.0 behaviour of PHP-CSS-Parser, where passing a Rule instance would
      * remove all rules with the same name. To get the old behaviour, use `removeRule($rule->getRule())`.
      *
-     * @param Rule|string|null $mRule
+     * @param Rule|string|null $removalPattern
      *        pattern to remove. If $mRule is null, all rules are removed. If the pattern ends in a dash,
```

Comment:
This mentions the parameter by name, and would also need updating.  Though could perhaps be modified to avoid being self-referential.